docs/source/support/release-notes.rst

```diff
@@ -6,6 +6,82 @@ Release notes
 
 This page contains release notes for the SDK.
 
+.. _release-0-13-42:
+
+0.13.42 - 2020-01-08
+--------------------
+
+JSON API - Experimental
+~~~~~~~~~~~~~~~~~~~~~~~
+
+- Rename ``argument`` in active contract to ``payload``. See #3826.
+- Change variant JSON encoding. The new format is ``{ tag: data-constructor, value: argument }``.
+  For example, if we have: ``data Foo = Bar Int | Baz``, these are all valid JSON encodings for
+  values of type Foo:
+  - ``{"tag": "Bar", "value": 42}``
+  - ``{"tag": "Baz", "value": {}}``
+  See #3622
+- Fix ``/contracts/lookup` find by contract key.
+- Fix ``/command/exercise`` to support any LF type as a choice argument.
+  See #3390
+
+
+DAML Compiler
+~~~~~~~~~~~~~
+
+- Move more types from daml-stdlib to standalone LF packages. The module names for the types have
+  also changed slightly. This only matters over the Ledger API when you specify the module name
+  explicitly. In DAML you should continue to use the existing module names.
+
+  - The types from ``DA.Semigroup` are now in a separate package under
+  ``DA.Semigroup.Types``
+  - The types from ``DA.Monoid` are now in a separate package under
+  ``DA.Monoid.Types``
+  - The types from ``DA.Time` are now in a separate package under
+  ``DA.Time.Types``
+  - The types from ``DA.Validation` are now in a separate package
+  under ``DA.Validation.Types``
+  - The types from ``DA.Logic` are now in a separate package under
+  ``DA.Logic.Types``
+  - The types from `DA.Date` are now in a separate package under
+  `DA.Date.Types`.
+  - The `Down` type from `DA.Internal.Prelude` is now in a separate
+  package under `DA.Internal.Down`.
+
+
+DAML SDK
+~~~~~~~~
+
+- ``daml damlc docs`` now accepts a ``--exclude-instances`` option to exclude unwanted instance docs
+  by class name.
+
+DAML-ON-X-SERVER
+~~~~~~~~~~~~~~~~
+
+- Made ledger api server to bind to localhost by default instead to the public
+  interface for security reasons.
+
+DAML Assistant
+~~~~~~~~~~~~~~
+
+- Bash completions for the DAML assistant are now available via ``daml install``. These will be
```

Comment:
This is missing the ZSH completions which have been merged in the meantime.

Reply:
Is it enough to just add `Bash/Zsh`? Seems there was no change log about this.

Reply:
There is a changelog but it was probably merged after you generated this https://github.com/digital-asset/daml/commit/d1449e78c7faa80bdaf5496f3585bbd8c44d4cba

Reply:
ok, rebased and included.